### ISSUE TITLE
Improving image-sentence embeddings using large weakly annotated photo collections

### ISSUE THREAD

**Issue by sweaterr** [sweaterr/deep-learning-readinglist#80]

**Comment by sweaterr** [comment]
Abstract. This paper studies the problem of associating images with
descriptive sentences by embedding them in a common latent space. We
are interested in learning such embeddings from hundreds of thousands
or millions of examples. Unfortunately, it is prohibitively expensive to
fully annotate this many training images with ground-truth sentences.
Instead, we ask whether we can learn better image-sentence embeddings
by augmenting small fully annotated training sets with millions of images
that have weak and noisy annotations (titles, tags, or descriptions).
After investigating several state-of-the-art scalable embedding methods,
we introduce a new algorithm called Stacked Auxiliary Embedding that
can successfully transfer knowledge from millions of weakly annotated
images to improve the accuracy of retrieval-based image description

**Comment by sweaterr** [comment]
이미지 피쳐
* CNN

we will refer as a “document” to each separate piece of text associated with an image: a sentence, a title, a set of tags, or a description.

Surprisingly, titles achieve the highest performance despite having the shortest average length.

descriptions of the Flickr images are by far the longest, but their predictive power is the worst.

In the end, combining all three document types achieves the best performance, so in the following experiments, we will use the combined text feature for Flickr1M and SBU1M.

**Comment by sweaterr** [comment]
이미지
![image](https://cloud.githubusercontent.com/assets/1518919/26675810/44d202d4-4700-11e7-82a4-00f65b47b0be.png)

텍스트
![image](https://cloud.githubusercontent.com/assets/1518919/26675832/50dfc264-4700-11e7-9e48-0be59df8986f.png)

모델
![image](https://cloud.githubusercontent.com/assets/1518919/26675873/7432420a-4700-11e7-8648-94979d4151ab.png)

![image](https://cloud.githubusercontent.com/assets/1518919/26675888/7cbe166a-4700-11e7-942e-3793766773d2.png)

 as XW and Y U to a common c dimensional latent space
